### PR TITLE
Created a program to find the day of the date.

### PR DESCRIPTION
**Pull request by Mayank662**
In this program I calculate the day of the week for any given date of calendar starting from 01/01/2000 to 31/12/2100 without using any inbuilt python function or library.

### CODE REVIEW

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
@Mayank662  Include your algorithm inside a function and call it from main function.
There are too many if else conditions instead of that u can follow switch casing in python which will improve readability of code.
Keep in mind, to follow PEP8 Guidelines closely for naming conventions.
https://www.python.org/dev/peps/pep-0008/

**Comment by Mayank662** [comment]
@SKAUL05  Thanks for the review, I have made changes as you have mentioned

File: Basic-Scripts/day_of_date.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#Function to calculate day of the week
+def dayofweek(
+        d, m, y):
+    
+    t = [
```
Comment (SKAUL05) [inline Basic-Scripts/day_of_date.py:5]: Use suitable and meaningful variable names instead of alphabets.

File: Basic-Scripts/day_of_date.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#Function to calculate day of the week
+def dayofweek(
```
Comment (SKAUL05) [inline Basic-Scripts/day_of_date.py:2]: Include Docstrings for function definition and follow naming conventions for functions.

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**